test/parallel/test-global.js

```diff
@@ -34,17 +34,21 @@ baseFoo = 'foo'; // eslint-disable-line no-undef
 global.baseBar = 'bar';
 
 assert.strictEqual(global.baseFoo, 'foo',
-                   'x -> global.x in base level not working');
+                   // eslint-disable-next-line no-undef
+                   `${baseFoo} -> global.baseFoo in base level not 
+                   working`);
 
 assert.strictEqual(baseBar, // eslint-disable-line no-undef
                    'bar',
-                   'global.x -> x in base level not working');
+                   // eslint-disable-next-line no-undef
+                   `${global.baseBar} -> baseBar in base level not 
```

Comment:
This too will be very confusing:  "bar -> baseBar in base level not working"....huh?

Try something like this instead:

```js
`global.x -> x failed: baseBar = ${baseBar}`
```